search_service/__init__.py

```diff
@@ -74,6 +74,7 @@ def create_app(*, config_module_class: str) -> Flask:
     api_bp.add_url_rule('/healthcheck', 'healthcheck', healthcheck)
     api = Api(api_bp)
     # Table Search API
+    api.add_resource(SearchTableFilterAPI, '/search_table')
```

Comment:
I think once we have dashboard, we should rename this endpoint to be more generic with accepting a dataset type given it is using query string. (maybe a todo comment)

Reply:
Agreed.